### ISSUE TITLE
nova: better error message on not-quite-right FITS images (or fix fits2fits.py)

### ISSUE THREAD

**Issue by dstndstn** [dstndstn/astrometry.net#39]
FITS images with, eg, invalid header cards are often encountered, and currently we fail hard on them.

**Comment by davidwhogg** [comment]
Can we "handle" this exception -- and call out to fitsverify and then say "image fails to fitsverify" and maybe give some further diagnostics?

**Comment by dstndstn** [comment]
yeah... I think the problem is that 'fits2fits.py' is broken -- it's
supposed to be able to fix FITS images with slight issues.  But I think
it's choking.
